src/Codeception/Test/Unit.php

```diff
@@ -92,6 +93,20 @@ public function getModule(string $module): Module
         return $modules[$module];
     }
 
+    /**
+     * Starts interactive pause in this test
+     *
+     * @param array<string, mixed> $vars
+     * @return void
+     */
+    public function pause(array $vars = []): void
+    {
+        $psy = (new PauseShell())->getShell();
```

Comment:
`codecept_debug` and traint only pause in debug mode, this method pauses always.

I think that it should check debug mode too for consistency:
```php
if (!Debug::isEnabled()) {
    return;
}
```

Reply:
Added